frontend/src/test_file.py

```diff
@@ -0,0 +1 @@
+fggefwqk    ;al
```

Comment:
@kimia84 happy to approve everything, whats this though?

Reply:
It seems like the file was pushed by mistake - I have deleted it